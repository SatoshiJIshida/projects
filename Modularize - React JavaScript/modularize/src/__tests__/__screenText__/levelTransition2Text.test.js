import React from "react";
import { shallow } from "enzyme";
import LevelTransition2Text from "../../components/screenText/levelTransition2Text";

/**
 * Library Jest taken from source https://jestjs.io/
 * Library Enzyme taken from source https://www.npmjs.com/package/enzyme
 */

describe("Level Transition Text (2)", () => {
  it("should be defined", () => {
    expect(LevelTransition2Text).toBeDefined();
  });

  it("should render correctly", () => {
    const levelTransition2Text = shallow(
      <LevelTransition2Text name="levelTransition2Text-test" />
    );
    expect(levelTransition2Text).toMatchSnapshot();
  });
});
