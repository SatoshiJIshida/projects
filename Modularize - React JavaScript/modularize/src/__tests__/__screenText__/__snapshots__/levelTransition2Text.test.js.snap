// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Level Transition Text (2) should render correctly 1`] = `ShallowWrapper {}`;
